packages/waku/src/lib/builder/build.ts

```diff
@@ -203,6 +204,7 @@ const buildServerBundle = async (
           ...serverEntryFiles,
         },
         output: {
+          format: serve === 'aws-lambda' ? 'esm' : 'es',
```

Comment:
@dai-shi please check if 'es' is the correct default value for all other build targets

Reply:
es and esm are same, and esm is not the valid option

Reply:
What's the default format if we don't specify anything?

Reply:
sorry  for this :-( I removed it and I still was able to bundle a working app for aws lambda